ida-udf-ws/src/main/java/upb/ida/controllers/sparqlQueryController.java

```diff
@@ -0,0 +1,37 @@
+package upb.ida.controllers;
+
+import org.apache.jena.query.*;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.util.FileManager;
+
+import org.springframework.web.bind.annotation.*;
+
+import java.io.ByteArrayOutputStream;
+
+@RestController
+@RequestMapping("/sparkle")
+public class sparqlQueryController {
```

Comment:
Hi @AsjadLFC good work 👍 
But why you put `@RequestMapping` before `class` is it intentional?

Reply:
yes, I was doing it for testing before but will remove it, it seems unnecessary